ephyviewer/datasource/neosource.py

```diff
@@ -175,23 +118,41 @@ def __init__(self, neorawio, channel_indexes=None, stream_index=None):
 
         self.neorawio = neorawio
 
-        if stream_index is not None:
-            self.stream_index = stream_index
-        elif self.neorawio.signal_streams_count() == 1:
-            self.stream_index = 0
-        else:
-            raise ValueError(f'Because the Neo RawIO source contains multiple signal streams ({self.neorawio.signal_streams_count()}), stream_index must be provided')
-
         if channel_indexes is None:
             channel_indexes = slice(None)
         self.channel_indexes = channel_indexes
 
-        self.stream_id = self.neorawio.header['signal_streams'][self.stream_index]['id']
-        signal_channels = self.neorawio.header['signal_channels']
-        mask = signal_channels['stream_id'] == self.stream_id
-        self.channels = signal_channels[mask][self.channel_indexes]
+        if V(neo.__version__)>='0.10.0':
+            # Neo >= 0.10
+            # - versions 0.10+ index channels within a stream
+            if stream_index is not None:
+                self.stream_index = stream_index
+            elif self.neorawio.signal_streams_count() == 1:
+                self.stream_index = 0
+            else:
+                raise ValueError(f'Because the Neo RawIO source contains multiple signal streams ({self.neorawio.signal_streams_count()}), stream_index must be provided')
+            self.stream_id = self.neorawio.header['signal_streams'][self.stream_index]['id']
+            signal_channels = self.neorawio.header['signal_channels']
+            mask = signal_channels['stream_id'] == self.stream_id
+            self.channels = signal_channels[mask][self.channel_indexes]
+        else:
+            # Neo < 0.10
```

Comment:
We youu add here `assert stream_index is None`

Reply:
Good idea.